app/Support/Modules/DatabaseActivator.php

```diff
@@ -0,0 +1,136 @@
+<?php
```

Comment:
This looks awesome!!

Reply:
Thanks!! 👯‍♂️

Reply:
I hope other things are resolved as well?
Or shall I make more changes?